docs/design/lexical_conventions/comments.md

```diff
@@ -0,0 +1,156 @@
+    contents of a block string literal to be commented out. This alternative has
+    the disadvantage that it would result in surprising behavior inside string
+    literals containing Carbon code.
+-   Fully lexed block comments: Here a token sequence between the opening and
+    closing comment marker is produced and discarded, with the lexing rules
+    relaxed somewhat to avoid rejecting ill-formed code. This would be analogous
+    to C and C++'s `#if 0` ... `#endif`. This alternative has the disadvantage
+    that it would be unable to cope with incomplete code fragments, such as an
+    unterminated block string literal. Its also inefficient to process compared
+    to non-lexing syntaxes.
+-   A hybrid approach: Using `//\{` and `//\}` delimiters that are invalid in
+    non-raw string literals, and with an indentation requirement for raw string
+    literals only. This alternative has the disadvantage of introducing
+    additional complexity into the lexical rules by treating different kinds of
+    string literals differently.
+-   Use of `/*` and `*/` as comment markers. This alternative has the  
+    disadvantage that it risks confusion by using similar syntax to C and C++
+    but with divergent semantics.
+
+After considerations, it was decided not to provide support to block comments.
+
+### Documentation comments
+
+A distinct comment syntax for documentation comments, there by treating
+documentation comments as producing real tokens rather than being stripped out
+by the lexer. A significant support for using a syntax that does not resemble a
+comment for representing documentation. For example, we could introduce an
+attribute syntax, such as using `@ <expression>` as a prefix to a declaration to
+attach attributes. Then a string literal attribute can be treated as
+documentation:
+
+```carbon
+@"Get the size of the thing."
+fn GetThingSize() -> Int;
+@"""
+Rate the quality of the widget.
+Returns a quality factor between 0.0 and 1.0.
+"""
+fn RateQuality(
+  @"The widget to rate."
+  Widget: w,
+  @"A widget quality database."
+  QualityDB: db) -> Float;
+```
+
+This use case will be explored by a FUTURE PROPOSAL.
+
+### Code folding comments
+
+Its found that some of the code editors can "fold" regions of a source file in
+order to ease navigation. In some cases, these fold regions can be customized by
+the use of comment lines.
+
+Example: In VS Code, this is accomplished with comments containing `#region` and
+`#endregion`:
+
+```
+// #region Functions F and G
+fn f() { ... }
+fn g() { ... }
+// #endregion
+```
+
+Supporting such markers as normal text within line comments requires no
+additional effort, but consider introducing a specific Carbon syntax for region
+comments. This would encourage a common representation across code editors. This
+would be handled by a new form of comment in the future.
+
+## References
+
+-   Proposal
+    [#198 Comments](https://github.com/carbon-language/carbon-lang/pull/198)
```

Comment:
```suggestion
    [#198: Comments](https://github.com/carbon-language/carbon-lang/pull/198)
```

Reply:
Added it!

Reply:
Still not fixed

Reply:
Had a couple of issues with pre-commit! I indeed added it first but then had to restart everything from scratch and, yeah missed it! Will add it now!

Reply:
Done!